tesseract_python/tesseract_python/package.xml

```diff
@@ -16,11 +16,15 @@
   <depend>tesseract_common</depend>
   <depend>tesseract_visualization</depend>
 
-  <depend>python</depend>
-  <depend>python-numpy</depend>  
+  <depend condition="$ROS_PYTHON_VERSION == 2">python</depend>
```

Comment:
still `<package format="2">`

Reply:
@ipa-mdl Ah. I had CATKIN_IGNORE in this package locally so everything I was testing locally was working. Sorry about that. Thank you for the help.